roles/upspin/tasks/start.yml

```diff
@@ -10,5 +11,10 @@
         file: ../vars/config_vars.yml
         name: config
 
-    - name: Start tasks
-      shell: "upspin setupserver --host={{ config.domain }} --domain={{ config.domain }}"
\ No newline at end of file
+    - name: Start Upspin server
+      shell: upspin setupserver --host="{{config.domain}}.{{config.zone}}" --domain="{{config.domain}}.{{config.zone}}"
+      register: result
+      until: result.rc == 0
+      retries: 10
+      delay: 2
+      ignore_errors: yes
```

Comment:
@patrickdillon Here is one dumb solution I came up with. It seems that enabling the service takes some time and we continuously retry the `setupserver` command until it gets to work. After tested, it worked well but maybe it is not a good solution for this kind of bug.